lib/adzerk/advertiser.rb

```diff
@@ -6,7 +6,7 @@ def search(advertiser_name)
       parse_response(client.post_request(url, data))
     end
 
-    def instant_counts(advertiser_id)
+    def instant_counts(advertiser_id, page: 1, pageSize: 500)
```

Comment:
Instant Counts aren't paged. We should remove these from the URL rather than add parameters.

Reply:
Updated